src/lib/edge-functions/bootstrap.mjs

```diff
@@ -1,5 +1,5 @@
 import { env } from 'process'
 
-const latestBootstrapURL = 'https://64e7783fce8cfe0008496c72--edge.netlify.com/bootstrap/index-combined.ts'
+const latestBootstrapURL = 'https://deploy-preview-294--edge.netlify.app/bootstrap/index-combined.ts'
```

Comment:
Let's not forget to change this before merging.

Reply:
done in https://github.com/netlify/cli/pull/5968/commits/bb5f238fcd8ce2926679167ec1daf84844968490